functions/New-DbaAgentJob.ps1

```diff
@@ -0,0 +1,476 @@
+                    $Job.NetSendLevel = $NetsendLevel
+
+                    Write-Message -Message "Setting job netsend operator" -Level 5 -Silent $Silent
+                    $Job.OperatorToNetSend = $NetsendOperatorName
+                }
+                else 
+                {
+                    Stop-Function -Message ("The netsend operator name '$($NetsendOperatorName)' does not exist on instance '$($SqlServer)'. Exiting..") -Silent $Silent -Target $JobName
+                    return
+                }   
+            }  
+            else 
+            {
+                Write-Message -Message "Invalid combination of netsend operator name '$($NetsendOperatorName)' and netsend level $($NetsendLevel). Not setting the notification." -Warning -Silent $Silent
+            }        
+        }
+
+        if($PageOperatorName.Length -ge 1)
+        {
+            if($PageLevel -ge 1)
+            {
+                # Check if the operator name is present
+                if(($Server.JobServer.Operators).Name -contains $PageOperatorName)
+                {
+                    Write-Message -Message "Setting job pager level" -Level 5 -Silent $Silent
+                    $Job.PageLevel = $PageLevel
+
+                    Write-Message -Message "Setting job pager operator" -Level 5 -Silent $Silent
+                    $Job.OperatorToPage = $PageOperatorName
+                }
+                else 
+                {
+                    Stop-Function -Message ("The page operator name '$($PageOperatorName)' does not exist on instance '$($SqlServer)'. Exiting..") -Silent $Silent -Target $JobName
+                    return
+                }
+            }
+            else 
+            {
+                Write-Message -Message "Invalid combination of page operator name '$($PageOperatorName)' and page level $($PageLevel). Not setting the notification." -Warning -Silent $Silent
+            }
+        }
+
+        if($DeleteLevel -ge 0)
+        {
+            Write-Message -Message "Setting job delete level" -Level 5 -Silent $Silent
+            $Job.DeleteLevel = $DeleteLevel
+        }
+        #endregion job options
+
+        # Execute 
+        if($PSCmdlet.ShouldProcess($SqlServer, ("Creating the job $($SqlServer)"))) 
+        {
+            try
+            {
+                Write-Message -Message ("Creating the job") -Level 2 -Silent $Silent
+                
+                # Create the job
+                $Job.Create()
+
+                Write-Message -Message "Job created with UID '$($Job.JobID)'" -Level 5 -Silent $Silent
+            }
+            catch
+            {
+                Write-Message -Message ("Something went wrong creating the job. `n$($_.Exception.Message)") -Level 2 -Silent $Silent 
+                
+            }
+        }
+        
+        # Add the job to the list
+        $JobsCreated += $Job
+    }
+
+    END
+    {
+        Write-Message -Message "Finished creating job(s)." -Level 2 -Silent $Silent
```

Comment:
Please consider named levels i the future - like Output, Verbose, VeryVerbose, etc.